archunit/src/main/java/com/tngtech/archunit/lang/syntax/elements/ClassesShouldConjunction.java

```diff
@@ -34,7 +34,7 @@
  * </code></pre>
  *
  * The semantics of the new condition will be {@code (public OR private) AND haveNameMatching}
- * – and not {@code public || (private && haveNameMatching)}.
+ * - and not {@code public || (private && haveNameMatching)}.
```

Comment:
Would this work for you as well?
```diff
--- a/buildSrc/src/main/groovy/archunit.java-conventions.gradle
+++ b/buildSrc/src/main/groovy/archunit.java-conventions.gradle
@@ -15,4 +15,5 @@ tasks.withType(JavaCompile) { Task task ->
 
 javadoc {
     options.addBooleanOption('html5', true)
+    options.encoding = UTF_8.toString()
 }
```

Reply:
Yes, it works too, I reverted my change and applied your suggestion.